scripts/generate-analysis-files-repo.sh

```diff
@@ -27,14 +27,14 @@ mkdir -p ${compiled_dir}
 
 # Create the independent sample list using the *FULL* histology file
 echo "Create independent sample list"
-bash ${analyses_dir}/independent-samples/run-independent-samples.sh
+bash ${analyses_dir}/independent-samples-release/run-independent-samples.sh
```

Comment:
```suggestion
bash ${analyses_dir}/independent-samples-release/run-independent-samples-release.sh
```

An optional enhancement is to rename `run-independent-samples.sh` to `run-independent-samples-release.sh`, to be consistent with module name.